app/Console/Commands/CampaignVisibileEntityCount.php

```diff
@@ -57,8 +58,14 @@ public function handle()
                 $campaign->save();
             }
         });
-
-        $this->info('Updated ' . $this->count . ' public campaigns.');
+        $log = "Updated {$this->count} public campaigns.";
+        $this->info($log);
+        if (config('app.log_jobs')) {
+            JobLog::create([
+                'name' => 'campaign-visible-entity-count',
```

Comment:
Can you everywhere replace `'name' => '...'` with `'name' => $this->signature`? I think that will be easier to read and then look up in the code :)